source/components/tooltip/tooltip-docs.less

```diff
@@ -1 +1,32 @@
-// TBD
+@import 'vars';
+
+.tooltip-examples {
+  hx-tooltip {
+    display: inline-block;
+    margin: 1em;
+    position: relative;
+  }
+}
+
+.demo-target {
+  border: 3px dotted #000000;
+  margin: auto;
+  min-width: 200px;
+  padding: 10px;
+  text-align: center;
+  width: 50%;
```

Comment:
Some of these aren't needed, it could just be:
```
.demo-target {
  border: 3px dotted @gray-1000;
  padding: 10px;
  text-align: center;
  hx-icon {
    color: @gray-700;
    vertical-align: text-top;
  }
}
```